solutions/loading-web-fonts/package.json

```diff
@@ -9,8 +9,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@next/font": "^13.0.2",
     "@vercel/examples-ui": "^1.0.4",
-    "next": "latest",
+    "next": "canary",
```

Comment:
Why?

Reply:
`canary` is never supposed to have breaking changes and so we set examples to it to have them with the latest changes when cloned, this is all optional and we have had cases where using `canary` would be needed to fix a bug, and also where a breaking change was introduced accidentally by Next.js, and this helps catching that.